plugin/src/main/java/org/opensearch/ml/sdkclient/util/JsonTransformer.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.ml.sdkclient.util;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.annotations.VisibleForTesting;
+
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+public class JsonTransformer {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @VisibleForTesting
+    public static Map<String, AttributeValue> convertJsonObjectToItem(JsonNode jsonNode) {
```

Comment:
`convertJsonObjectToDDbAttributeMap`? as this method is only taking care of Dynamodb? Same goes for the other method name.

Reply:
Rename the class too? `JsonTransformer` sounds too generic in this case.

Reply:
Will update method name, however would like to keep the name generic to support other json transformation functions in future